apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingDelegateChanges/StakingDelegateChanges.types.ts

```diff
@@ -0,0 +1,14 @@
+export type V2StakingDelegateChangeItem = {
```

Comment:
```suggestion
export type StakingDelegateChangeItem = {
```
No need to have `V2` in the name, it's an unnecessary implementation detail.

Reply:
Also, do you need this type if you just cast the result in `useGetStakingDelegateChanges` to it (so it means these 2 types are compatible)?